### PR TITLE
i10n Update for Occitan

### PR DESCRIPTION
**Pull request by Quent-in**
3 files changed with new strings

### CODE REVIEW

**Comment by Quent-in** [comment]
Well...one more time I don't understand the issue I have here =/

File: config/locales/oc.yml [hunk 1]
```diff
@@ -373,6 +437,76 @@ oc:
     click_to_show: Clicatz per afichar
     reblogged: a partejat
     sensitive_content: Contengut sensible
+terms:
```
Comment (ykzts) [inline config/locales/oc.yml]: add indent.

Comment (Quent-in) [inline config/locales/oc.yml]: Done.

File: config/locales/oc.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 oc:
   about:
-    about_mastodon: Mastodon es un malhum social <em>liure e open-source</em>. Una alternativa <em>decentralizada</em> a las platformas comercialas, aquò evita qu’una sola companiá monopolize vòstra comunicacion. Causissètz une servidor que vos fisatz, quina que siague vòstra causida, podètz interagir amb tot lo mond. Qual que siague pòt aver son instància Mastodon e participar al <em>malhum social</em> sens cap de problèmas.
+    about_mastodon: Mastodon es un malhum social <em>liure e open-source</em>. Una alternativa <em>descentralizada</em> a las plat-formas comercialas, aquò evita qu’una sola companhiá monopolize vòstra comunicacion. Causissètz un servidor que vos fisatz, quina que siasque vòstra causida, podètz interagir amb tot lo mond. Qual que siasque pòt aver son instància Mastodon e participar al <em>malhum social</em> sens cap de problèmas.
```
Comment (ykzts) [inline config/locales/oc.yml]: `about_mastodon` -> [`about_mastodon_html`](https://github.com/tootsuite/mastodon/blob/4d661e11837c12a3381a08cb686fc5451e8d7deb/config/locales/en.yml#L4)

Comment (Quent-in) [inline config/locales/oc.yml]: Changed

Comment (ykzts) [inline config/locales/oc.yml]: The content of the message body has changed greatly in English. Is it unnecessary to change it even with `oc`?

**Review by Gargron — APPROVED** [review]

File: config/locales/oc.yml (hunk 2 above)
Comment (Quent-in) [inline config/locales/oc.yml]: True, I didn't notice, it's way shorter now. I'll rephrase it in another PR when I have some spare time. Thanks for the observation @ykzts !

**Pull request merged**